packages/components/pie-webc-core/src/test-helpers/percy-lit-options.ts

```diff
@@ -0,0 +1,37 @@
+///*
+///* This code snippet was provided by Percy as a way to get our Lit components working correctly when running against the Safari browser.
+///* Safari blocks requests coming from localhost, resulting in our CSS not loading when requested.
+///* This function is passed to Percy and executed against the serialised DOM to rewrite all 'localhost' urls to 'render.percy.local'.
+///* This is a temporary workaround, and a long-term solution will be implemented by Percy at a later date.
+///*
+const percyOptions = {
+    domTransformation: (documentElement: any) => {
+        function updateLinks(root : any) {
+        root.querySelectorAll('[data-percy-adopted-stylesheets-serialized]').forEach((link : any) => {
+            console.log(link);
```

Comment:
Forgotten console.log? Or are we leaving it deliberately?

Reply:
@sebastienrocks replied in the other comment :)